src/components/multi-select-control/index.js

```diff
@@ -19,10 +20,11 @@ const defaultProps = {
 		label: '',
 		value: [],
 	},
+	isSearchable: false,
 	setAttributes: () => {},
 };
 
-export default function UAGMultiSelectControl( { label, options, data, setAttributes } ) {
+export default function UAGMultiSelectControl( { label, options, data, isSearchable, setAttributes } ) {
```

Comment:
Since we now have more props, and chances are we'll need to add more in the future, we can do it like this.

```suggestion
export default function UAGMultiSelectControl( props ) {
	const {
		label,
		options,
		data,
		isSearchable,
		setAttributes,
	} = props
	
```

Reply:
done